playground/backend/cmd/server/controller.go

```diff
@@ -134,6 +137,35 @@ func (controller *playgroundController) GetRunOutput(ctx context.Context, info *
 	return &pipelineResult, nil
 }
 
+// GetLogs is returning logs of execution for specific pipeline by PipelineUuid
+func (controller *playgroundController) GetLogs(ctx context.Context, info *pb.GetLogsRequest) (*pb.GetLogsResponse, error) {
+	errorTitle := utils.GetFuncName(controller.GetRunOutput)
+	pipelineId, err := uuid.Parse(info.PipelineUuid)
+	if err != nil {
+		logger.Errorf("%s: %s: pipelineId has incorrect value and couldn't be parsed as uuid value: %s", info.PipelineUuid, errorTitle, err.Error())
+		return nil, errors.InvalidArgumentError(errorTitle, "pipelineId has incorrect value and couldn't be parsed as uuid value: "+info.PipelineUuid)
+	}
+	lastIndex, err := code_processing.GetLastIndex(ctx, controller.cacheService, pipelineId, cache.LogsIndex, errorTitle)
```

Comment:
if lastIndex, err := .... ; err != nil {
return nil, err
}

Reply:
in that case, I can't use `lastIndex` outside of `if` part.

Reply:
Oh, ok